ksp_plugin/plugin.hpp

```diff
@@ -427,10 +424,25 @@ class Plugin {
 
   not_null<std::unique_ptr<PhysicsBubble>> const bubble_;
 
-  // |bodies_| and |initial_state_| are null if and only if |!initializing_|.
-  // TODO(egg): optional.
-  std::unique_ptr<IndexToMassiveBody> bodies_;
-  std::unique_ptr<IndexToDegreesOfFreedom> initial_state_;
+
+  struct AbsoluteInitializationObjects {
+    IndexToMassiveBody bodies;
+    IndexToDegreesOfFreedom initial_state;
+  };
```

Comment:
I'd very much like the whole thing to be a variable of an unnamed struct, but then it wouldn't be optional (you can't declare a struct within a template parameter).